### ISSUE TITLE
[Firefox] Alt+F10 keystroke to focus the toolbar does not work in fullscreen

### ISSUE THREAD

**Issue by oleq** [ckeditor/ckeditor5#18247]
## 📝 Provide detailed reproduction steps (if any)

1. Go to fullscreen docs.
2. Focus the editor.
3. Use Alt+F10 (focus moves to the toolbar)
4. Enter fullscreen
5. Use Alt+F10 

### ✔️ Expected result

Focus should also move to the toolbar.

### ❌ Actual result

It does not. It works in Chrome but not in Firefox. Alt+F9 works (focuses menu bar).

## 📃 Other details

* Browser: 136.0.3 (aarch64)
* OS: MacOS

**Comment by oleq** [comment]
Actually, it's even weirder than that. Touching the toolbar in the screen mode brings back the keystroke support

https://github.com/user-attachments/assets/2fdcf6db-b745-4a75-8838-04bcc042cc32

**Comment by Dumluregn** [comment]
Done in https://github.com/ckeditor/ckeditor5/pull/18273.

**Comment by lkszzajac** [comment]
The issue still persists:
- `Alt + F10` works only if FS is entered via `CMD + Shift + F` shortcut
- does not work at all after exiting the FS
- does not work if FS is entered via the toolbar button

Recording:

https://github.com/user-attachments/assets/6a10b330-c181-494e-bb29-3e4da4d7d2bf

**Comment by scofalik** [comment]
Fixed on `release`.
